changelog/unreleased/fix-docker

```diff
@@ -0,0 +1 @@
+Fix Docker configuration for DISABLE_AUTHENTICATION
```

Comment:
Thanks for adding a changelog entry, this is totally needed for this change. However, please use the template file at https://github.com/restic/rest-server/blob/master/changelog/TEMPLATE (copy it into this file and edit the text), and also rename this file to "issue-119".

Reply:
Sorry, I think I've resolved it.